### DISCUSSION TITLE
favicon not showing

### DISCUSSION THREAD

**Discussion by sufiprog** [vercel/next.js#70748] [Help] [answered]
### Summary

why favicon not showing in next 15 however i write the correct syntax
![s](https://github.com/user-attachments/assets/89f13f0c-f533-49e2-a2f3-d13dfd912457)

**Comment by icyJoseph** [comment, marked as answer]
Make sure to purge your browser cache, try incognito mode, or another browser. Favicons are cached for a while, and often fail to show that they are updated.

- https://github.com/vercel/next.js/discussions/65973#discussioncomment-9495267
